podcast/the-changelog-330.md

```diff
@@ -78,15 +78,15 @@ That is super-useful, because basically the whole idea is that it brings visibil
+The cool thing about being SQL -- because I was, and I'm still thinking about offering a GraphQL thing, because Git repositories are trees, and once you parse code, you get a tree; so everything's trees, and GraphQL for trees is great... But the fact that it's SQL, it allows you to mix it with other datasets. You have Looker, or Power BI, or things like this where you can have many datasets and do a query across many different databases. Imagine doing something where you're saying -- say inner source; The whole goal under inner source is really make sure you break the silos in a company and that everybody collaborates with each other. Like the Google-style, Facebook-style, even though the inner source term  \[unintelligible 00:14:51.00\] at PayPal. In order to measure how well you're doing, the whole idea is that you need to first know who is in each team. Unfortunately, that is not in your Git dataset, so you're gonna need to mix it with some other dataset - HR dataset, or whatever it is. So Looker, or Power BI, and I think that even Tableau - they will allow you to do these kinds of things...
 
 **Adam Stacoviak:** You can look up the repo URL on GitHub, even if it's a private GitHub repo as well.
 
 **Francesc Campoy:** Yeah, yeah. So the cool thing is that--
 
 **Adam Stacoviak:** Because you have teams at the org level, so you could look up not so much by the repo, but by the repo URL.
 
-**Francesc Campoy:** Yeah, yeah. So the thing is that all of that -- that is the GitHub API. We work with any Git repository, so all of the concepts that we work with are Git, for now... That's why, for instance, the organization is a GitHub or GitLab \[unintelligible 00:15:39.12\] you could expose it from a different dataset. Just download the whole thing, put it in a MySQL and that's it. You can do that, too.
+**Francesc Campoy:** Yeah, yeah. So the thing is that all of that -- that is the GitHub API. We work with any Git repository, so all of the concepts that we work with are Git, for now... That's why, for instance, the organization is a GitHub or GitLab. Point you could expose it from a different dataset. Just download the whole thing, put it in a MySQL and that's it. You can do that, too.
```

Comment:
Maybe give this one another listen? `Point you could expose` doesn't make much sense when I read it? Could be what he said, but...

Reply:
I took a look at this, does sound like it's what he's saying. I've formatted a bit better
> "Point: you could expose it..."